packages/actor-init-query/lib/ActorInitQuery.ts

```diff
@@ -4,14 +4,14 @@ import type { IActionInit, IActorOutputInit } from '@comunica/bus-init';
 import { KeysInitQuery } from '@comunica/context-entries';
 import type { ICliArgsHandler, IQueryContextCommon } from '@comunica/types';
 import type { Readable } from 'readable-stream';
-import yargs from 'yargs';
 import type { IActorInitQueryBaseArgs } from './ActorInitQueryBase';
 import { ActorInitQueryBase } from './ActorInitQueryBase';
 import { CliArgsHandlerBase } from './cli/CliArgsHandlerBase';
 import { CliArgsHandlerQuery } from './cli/CliArgsHandlerQuery';
 import { QueryEngineBase } from './QueryEngineBase';
 
 const streamifyString = require('streamify-string');
+const yargs = require('yargs');
```

Comment:
This change is unnecessary I think, since the error can be fixed by adjusting the constructor argument. It looks like a typing issue, and converting it to require like this only makes the type `any` which works around the issue but loses types. I can make another PR for this.

Reply:
What would need to change exactly? If you can share that, @RubenEschauzier can try it on his end, and already include it in this PR.

Reply:
Changing the constructor argument to `[]` seems to fix it, with maybe also bumping the types version to the latest one. The functionality seems to remain unchanged.

Reply:
Can you try this @RubenEschauzier?

Also important: we don't have unit tests for the CLI tool, so could you have a quick manual check if `engines/query-sparql/bin/query.js` still seems to work after changing this?

Reply:
The error seems to be this:
```
packages/actor-init-query/lib/ActorInitQuery.ts:42:34 - error TS2345: Argument of type '{}' is not assignable to parameter of type 'string | readonly string[] |undefined'.
Type '{}' is not assignable to type 'undefined'.

42     let argumentsBuilder = yargs({});
```

Reply:
I could add the constructor changes to this PR if that is smarter?

Reply:
If that works, why not.

Reply:
The changes are added and I tested it with a manual check and it seems to still work.